assembl/models/post.py

```diff
@@ -631,12 +631,19 @@ def generate_message_id(cls):
         onupdate='CASCADE'
     ), primary_key=True)
 
+    modification_date = Column(DateTime)
```

Comment:
Why not use HistoryMixin? It provides tombstone date of the previous version, which corresponds to the last modification date.